download/nightly/index.md

```diff
@@ -37,6 +36,10 @@ a certain build is not available for your operating system, please check the pre
 #nightly-builds summary {
   cursor: pointer;
   display: list-item;
+  white-space: nowrap;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  width: 100%;
```

Comment:
I’m not sure whether I like this approach or not. When I originally implemented the truncation, it wasn’t really meant as a design choice but rather as a way to deal with people putting entire novels on the first line of their commit messages instead of splitting them into summary + body.

This CSS approach looks fine on a big-enough viewport, but if the width isn’t sufficient the commit message can potentially become quite short. I’m on my old WXGA notebook right now, and even just at half the screen width (e.g. dragging the browser window to one side of the screen) most lines are already truncated. And we *know* that we have a bunch of users on lower resolutions like that thanks to pencil2d/pencil#1417.

On the other hand, the 72 characters limit I had before works regardless of viewport width and was based on the convention for git commit messages, which is the same approach GitHub uses when truncating commit messages. It might even make sense to show the rest of the commit message when the entry of that nightly build is expanded (again, similar to how GitHub does it).

Reply:
I have adjusted this to display the full first commit line when the item has been expanded. I hope this is an acceptable compromise. I don't really like the idea of having them all wrap as that will just introduce a different problem on smaller resolution displays: it will make the list longer and more difficult to read being broken up by multiple lined messages.

Reply:
Well, I suppose it’s a choice between one evil or another. Seems like a fine compromise.